adapted-video-frontend/src/main/java/fr/labri/progress/comet/conf/RabbitMqConfiguration.java

```diff
@@ -17,7 +19,13 @@
 	@Bean
 	@Inject
 	public RabbitAdmin getAdmin(ConnectionFactory cf) {
-		return new RabbitAdmin(cf);
+		RabbitAdmin admin = new RabbitAdmin(cf);
+//		admin.declareQueue(new org.springframework.amqp.core.Queue("soft",
```

Comment:
please remove commented out lines (if we want to see the history, we'll use git)